novatel_gps_driver/test/novatel_gtest.test

```diff
@@ -0,0 +1,3 @@
+<launch>
+    <test test-name="novatel_gtest" pkg="novatel_gps_driver" type="novatel_gtest"/>
+</launch>
```

Comment:
This file isn't required to run the tests. You should be able to run your tests with catkin_make run_tests_novatel_gps_driver

Reply:
.test files are normally used for integration tests where multiple nodes are launched and the unit test actually communicates over the ros network.